libmscore/check.cpp

```diff
@@ -117,35 +117,33 @@ qDebug("checkScore: remove empty ChordRest segment");
                               }
 #if 0
                         if (cr->tick() > tick) {
-                              int ttick = tick;
-                              int ticks = cr->tick() - tick;
+                              Fraction ttick = tick;
+                              Fraction ticks = cr->tick() - tick;
 
-                              Fraction f = Fraction::fromTicks(ticks) / st->timeStretch(ttick);
+                              Fraction f = ticks / st->timeStretch(ttick);
                               qDebug("  insert %d/%d", f.numerator(), f.denominator());
 
-                              while (ticks > 0) {
+                              while (ticks > Fraction(0,1)) {
```

Comment:
Or make the default arguments in the constructor 0 and 1 and use `Fraction()`?

Reply:
The fastest way is to implement special function as i started with Fraction.isZero() like Fraction.ltZero() Fraction.leZero() etc. This functions evaluate to a simple integer compare without any (fraction) overhead . I will add them in addition to a user defined literal for Fractions so that you can write "if (ticks > 0_Fr) ...".

Reply:
Using "if (ticks > Fraction()) ..." is already possible but is IMHO unreadable bad style. Something like "if (ticks > Fraction(0)) ..." looks better and i added this as Fractions constructor.